src/lib/blockies/makeBlockiesUrl.ts

```diff
@@ -0,0 +1,57 @@
+import type { BlockiesOptions } from './blockies'
+import { buildOpts, createImageData } from './blockies'
+
+const store: Record<string, string> = Object.create(null)
+
+const _btoa =
+  btoa || window?.btoa || ((str: string) => Buffer.from(str, 'binary').toString('base64'))
+
+export const makeBlockiesUrl = function (
+  address: string,
+  size = 8,
+  caseSensitive = false,
+  scale = 10,
+) {
```

Comment:
LSP "convert function arguments to destructured objects" on all big arity blockies methods